src/tokens/ERC721.huff

```diff
@@ -104,6 +104,10 @@
 #define macro BALANCE_OF() = takes (0) returns (0) {
     NON_PAYABLE()                                       // []
     0x04 calldataload                                   // [account]
+    // revert if account is zero address
+    dup1 continue jumpi 
+    ZERO_ADDRESS(0x00)
+    continue:
     [BALANCE_LOCATION] LOAD_ELEMENT_FROM_KEYS(0x00)     // [balance]
     0x00 mstore                                         // []
     0x20 0x00 return                                    // []
```

Comment:
If you want to follow jtriley-eth's style guide (https://github.com/jtriley-eth/huff-style-guide#alignment)
```suggestion
    continue:
        [BALANCE_LOCATION] LOAD_ELEMENT_FROM_KEYS(0x00) // [balance]
        0x00 mstore                                     // []
        0x20 0x00 return                                // []
```

Reply:
I don't know exactly what huffmate's styling is, but it would lack continuity within the files if this commits. Maybe a issue should be opened?

Reply:
yes you are right, let's do it in an other proper PR

Reply:
#131